### PR TITLE
Branch/stereo r69

### PR DESCRIPTION
**Pull request by scottnc27603**
@allisoncorey @davideaster @BrettASwift 

All of the edits for version 69 of threejs are in this branch as well.  There are some additional material/shader/textrue modifications as well, so I switched the pull request over to this branch.

### CODE REVIEW

File: public/solarSystem/images/sky_stars.jpg [hunk 1]
```diff
@@ -0,0 +1,1316 @@
+    <!DOCTYPE HTML>
```
Comment (allisoncorey) [inline public/solarSystem/images/sky_stars.jpg]: Is this supposed to be a .jpg or .html file?

File: public/duck/index.vwf.config.yaml [hunk 2]
```diff
@@ -19,3 +19,4 @@ model:
   vwf/model/buzz:
```
Comment (allisoncorey) [inline public/duck/index.vwf.config.yaml]: Is the duck by default now rendered in stereo? I'm not seeing any difference when I run locally.

Do we want to have a separate app as a stereo example?

File: public/solarSystem/index.vwf.yaml [hunk 3]
```diff
@@ -0,0 +1,419 @@
...
+                    type: normalmap
+                    uniforms:
+                      tNormal: { type: 't', pValue: 'images/earth_normal_2048.jpg' }
+                      tDiffuse: { type: 't', pValue: 'images/earth_surface_2048.jpg' }
+                      tSpecular: { type: 't', pValue: 'images/earth_specular_2048.jpg' }
+                      enableSpecular: { type: 'i', pValue: true }
+                      enableDiffuse: { type: 'i', pValue: true }
+                    lights: true            
+
+                # working
```
Comment (allisoncorey) [inline public/solarSystem/index.vwf.yaml]: Is this commented section still necessary?

Comment (scottnc27603) [inline public/solarSystem/index.vwf.yaml]: No, it's been removed in b922724

File: public/solarSystem/images/sky_stars.jpg (hunk 1 above)
Comment (scottnc27603) [inline public/solarSystem/images/sky_stars.jpg]: This file was no longer needed, so it's been removed.

File: public/duck/index.vwf.config.yaml (hunk 2 above)
Comment (allisoncorey) [inline public/duck/index.vwf.config.yaml]: Just kidding... I see the stereo app now :)

**Comment by scottnc27603** [comment]
I'm working on a sample stereo application located in 'public/stereo'.  I hope to have that working soon.  All of the Smith's work was with Y up, so I'm not sure if the shaders will need to be modified.  It might be easier to just have this demo in y up.

**Comment by allisoncorey** [comment]
@scottnc27603 This looks good to me - I don't see any major issues. 

I set the duck application to stereo and it seemed to work. Do we have a way yet to configure offsets, or is that all in the StereoEffect.js file?

**Comment by scottnc27603** [comment]
We already do have a way to set the offsets, I just don't have an example of properties yet.  I guess I could put all of those values in the 'stereo' example?  How does that sound?

**Comment by scottnc27603** [comment]
:+1:

**Comment by allisoncorey** [comment]
Sounds good. Even if it's commented out, so you can see that's where they would be set.

Thanks! :+1:

**Pull request merged**